include/fastdds/rtps/transport/ChainingTransport.h

```diff
@@ -329,6 +329,11 @@ class ChainingTransport : public TransportInterface
             const fastrtps::rtps::Locator_t& local_locator,
             const fastrtps::rtps::Locator_t& remote_locator) = 0;
 
+    RTPS_DllAPI virtual void update_network_interfaces() override
```

Comment:
We don't need `virtual` when we have `override` or `final`.
Same elsewhere.

```suggestion
    RTPS_DllAPI void update_network_interfaces() override
```

Reply:
Done in 05d204f5f